tests/Mocks/IResearchLinkMock.cpp

```diff
@@ -39,7 +39,7 @@ namespace arangodb {
 namespace iresearch {
 
 IResearchLinkMock::IResearchLinkMock(IndexId iid, arangodb::LogicalCollection& collection)
-    : Index(iid, collection, IResearchLinkHelper::emptyIndexSlice()),
+    : Index(iid, collection, IResearchLinkHelper::emptyIndexSlice(0).slice()),
```

Comment:
it seems we pass `VPackSlice` to a temporary `VPackBuilder`, isn't it?

Reply:
No, it is safe as slice is never stored inside